code/data_structures/stack/balanced_expression/balanced_expression.java

```diff
@@ -0,0 +1,33 @@
+import java.util.*;
+
+class balanced_expression {
+  public static void main(String args[]) {
```

Comment:
Your indentation length seems inconsistent. Here you use 2 spaces, and below you use 2 tabs. Be consistent, and change the tabs to spaces. Everyone does not have the same idea of a tab length, so it can be confusing.